resolwe_bio/processes/import_data/seq_reads.yml

```diff
@@ -160,14 +161,53 @@
 
       mv fastqc/*_fastqc.zip .
 
-      for report_dir in ./fastqc/*/
-      do
-        ENCODING=`parse_encoding_type.py ${report_dir}fastqc_data.txt`
-        if [ "${ENCODING}" != "Sanger / Illumina 1.9" ]; then
-          re-error "Only Sanger / Illumina 1.9 encoding in supported."
-        fi
+      NAMES_FW=($FW_FASTQC)
+
+      for name in ${NAMES_FW[@]}; do
+        READS_NAME_FW=`basename $name _fastqc.zip`
+
+        for report_dir in ./fastqc/"${READS_NAME_FW}_fastqc"/; do
+          ENCODING=`parse_encoding_type.py ${report_dir}fastqc_data.txt`
+
+          if [ "${ENCODING}" == "Illumina 1.5" ] ||  [ "${ENCODING}" == "Illumina 1.3" ]; then
```

Comment:
This IF statement is the same as for the single ended reads. We should write a bash script that checks the encoding version and does the transformation. The we should call this script from both processes.